kubernetes-definitions/search/kafka-exporter.yaml

```diff
@@ -0,0 +1,47 @@
+# Service
```

Comment:
Do you applied this to gloud? Right now, there is no search kafka metrics in gcloud.

Reply:
Added to gcloud. Now you can see metrics for search topics.